src/main/java/org/opentripplanner/routing/impl/ATLFareServiceImpl.java

```diff
@@ -192,10 +198,19 @@ private static RideType classify(Ride ride) {
                     shortName.equalsIgnoreCase("101") ||
                     shortName.equalsIgnoreCase("102")) {
                     return RideType.COBB_EXPRESS;
+                } else if (shortName.equalsIgnoreCase("BLUE") ||
+                    shortName.equalsIgnoreCase("GREEN")) {
+                    return RideType.COBB_CIRCULATOR;
                 }
                 return RideType.COBB_LOCAL;
             case XPRESS_AGENCY_ID:
-                return RideType.XPRESS;
+                // Get hour of trip start
+                long hours = (ride.startTime / 60 / 60) % 24;
+                if(hours > 12) {
```

Comment:
Insert space after `if`.

Reply:
In practice, this should work, however afternoon (pm) in theory includes 12pm-1pm.

Reply:
That's true. I'll make it greater than or equal to